cranelift/codegen/src/ir/layout.rs

```diff
@@ -469,6 +469,17 @@ impl Layout {
     pub fn next_block(&self, block: Block) -> Option<Block> {
         self.blocks[block].next.expand()
     }
+
+    /// Mark a block as "cold". This will try to move it out of the
+    /// ordinary path of execution when lowered to machine code.
+    pub fn mark_cold_block(&mut self, block: Block) {
```

Comment:
More nitpicks: thoughts on naming this `set_cold`? To me `is_blah` and `set_blah` go together. Also `block` seems unnecessary since a block is the only argument here; it seems self evident.

Reply:
Yes, I like that better; changed.